packages/scss/src/theming/_palettes.scss

```diff
@@ -5,44 +5,69 @@ $palettes: (
 		"transparent": rgba(#FFCC00, .1),
 		"dark": darken(#FFCC00, 5%),
 		"darker": darken(#FFCC00, 20%),
-		"light": lighten(#FFCC00, 5%),
-		"lighter": lighten(#FFCC00, 20%)
+		"light": lighten(#FFCC00, 10%),
+		"lighter": lighten(#FFCC00, 20%),
+		"lightest": lighten(#FFCC00, 45%)
 	),
 	"secondary": (
 		"color": #FF6600,
 		"text": #FFFFFF,
 		"transparent": rgba(#FF6600, .1),
 		"dark": darken(#FF6600, 5%),
 		"darker": darken(#FF6600, 20%),
-		"light": lighten(#FF6600, 5%),
-		"lighter": lighten(#FF6600, 20%)
+		"light": lighten(#FF6600, 10%),
+		"lighter": lighten(#FF6600, 20%),
+		"lightest": lighten(#FF6600, 45%)
+	),
+	"poplee": (
```

Comment:
no palette figgo/poplee in the official release thx

Reply:
It's only for the review.
I'll remove them when @Clotilde will be ok with the new version of these new versions.